### PR TITLE
feat(PureScript): Add Purescript module

### PR DESCRIPTION
**Pull request by nobv**
#### Description
Added the module which displays the currently installed PureScript version.

#### Motivation and Context
Because there is no PureScript support yet.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Screenshots (if appropriate):
Screen Shot 2020-05-17 at 13 14 35

#### How Has This Been Tested?
- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:
- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -1380,3 +1381,28 @@ files = ["foo"]       # can specify filters
...
+| `symbol`   | `"<=> "`       | The symbol used before displaying the version of PureScript. |
+| `style`    | `"bold black"` | The style for the module.                                    |
+| `disabled` | `false`        | Disables the `purescript` module.                            |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[haskell]
```
Comment (erikprice) [inline docs/config/README.md:1406]: Should this be `[purescript]`?

File: src/modules/purescript.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+#[cfg(test)]
+mod tests {
+    use crate::modules::utils::test::render_module;
+    use ansi_term::Color;
+    use std::fs::File;
+    use std::io;
+
+    #[test]
+    fn folder_without_purscript_files() -> io::Result<()> {
```
Comment (erikprice) [inline src/modules/purescript.rs:43]: Should this be `folder_without_purescript_files`?

**Comment by nobv** [comment]
@erikprice 
Sorry, I mistook target of fixing and pushed.
So I did force push 🙇‍♂️

File: docs/config/README.md [hunk 3]
```diff
@@ -1380,3 +1381,28 @@ files = ["foo"]       # can specify filters
...
+
+- The current directory contains a `spago.dhall` file
+- The current directory contains a \*.purs files
+
+### Options
+
+| Variable   | Default        | Description                                                  |
+| ---------- | -------------- | ------------------------------------------------------------ |
+| `symbol`   | `"<=> "`       | The symbol used before displaying the version of PureScript. |
+| `style`    | `"bold black"` | The style for the module.                                    |
```
Comment (andytom) [inline docs/config/README.md:1398]: The default here is `bold black` but in the config it is `bold white` should this be changed?

File: src/modules/mod.rs [hunk 4]
```diff
@@ -81,6 +82,7 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         "nodejs" => nodejs::module(context),
         "package" => package::module(context),
         "php" => php::module(context),
+        "purescript" => purescript::module(context),
         "python" => python::module(context),
         "ruby" => ruby::module(context),
         "rust" => rust::module(context),
```
Comment (andytom) [inline src/modules/mod.rs:88]: Can you also add `purescript` to the description function below around line 134.

**Comment by andytom** [comment]
@nobv thanks for the contribution and @erikprice thank you for the review.

**Pull request merged**